client/ionic/src/pages/Menu.tsx

```diff
@@ -11,20 +11,20 @@ const Menu: React.FC = () => {
         <IonItem
           href="/protect-yourself"
           color="primary"
-          className="pb3 tc ph5 pt5"
+          className="pb3 tc ph4 pt2"
         >
           <IonLabel>Protect Yourself</IonLabel>
         </IonItem>
-        <IonItem href="/triage" color="primary" className="pb3 tc ph5">
+        <IonItem href="/triage" color="primary" className="pb3 tc ph4">
           <IonLabel>Check Your Health</IonLabel>
         </IonItem>
-        <IonItem href="/menu-distress" color="primary" className="pb3 tc ph5">
+        <IonItem href="/menu-distress" color="primary" className="pb3 tc ph4">
           <IonLabel>Feeling Distressed?</IonLabel>
         </IonItem>
-        <IonItem href="/local_maps" color="primary" className="pb3 tc ph5">
+        <IonItem href="/local_maps" color="primary" className="pb3 tc ph4">
```

Comment:
While we're here should we make this href kebab-case like the rest? (although I guess we're planning to remove local maps for initial release, so take this with a grain of salt)

Reply:
I guess that can be done in a future PR